npm/README.md

```diff
@@ -0,0 +1,24 @@
+# Setting up your computer to publish to npm
+
+Prerequisites: [Node](https://nodejs.org) with npm version 5.7.0+ (required for `npm ci` to work)
```

Comment:
Because of legacy projects, we ask folks to use `nvm` to install Node on their machines. Not sure if we want to have a note about that here or not. (cc @gerardo-rodriguez)

Reply:
@tylersticka Should I change the Node link to link to `nvm`, so people install it via that rather than from the Node website?

Reply:
That seems good to me!

This in particular: https://github.com/creationix/nvm#installation